src/driver/experimental_trial.py

```diff
@@ -212,6 +212,35 @@ def evaluate(self, seed: int) -> pd.DataFrame:
             ],
             axis=1,
         )
+        if self._trial_descriptor.modeling_strategy == "m3strategy":
+            union_inventory = self._dataset.maximum_reach
+            true_single_inventories = [p.max_reach for p in self._dataset._data]
+            predicted_single_inventories = [
+                c._max_reach for c in self.reach_surface._reach_curves
+            ]
+            result["union_inventory_reach"] = union_inventory
+            result["true_single_pub_inventory_reaches"] = [true_single_inventories]
```

Comment:
A list of a list?  Can this be represented in a DataFrame?  Maybe it is best not to include this.

Reply:
I'm trying to save the list `true_single_inventories` into the df.   Directly letting `result["true_single_pub_inventory_reaches"] = true_single_inventories` causes bug, since pandas would try to put the list `true_single_inventories` into multiple rows.  So I did `[true_single_inventories]` to put them in a row.

If it is not desirable to assign a list to a df cell, here're alternative off the top of my head:
(1) convert the list into a tuple.
(2) convert the list into str like '10, 10, 10'.
It is desirable to keep all info of the list for analysis of prediction error.

Reply:
My main concern is that I am not sure that we will be able to read and write this data to the CSV files that we are using for storing the intermediate DataFrames.  If this works, then I am fine with it as is.  Otherwise, you might need to convert it to a string list as you suggested.